types/office-js-preview/index.d.ts

```diff
@@ -2413,6 +2413,15 @@ declare namespace Office {
          * Triggers when a Resource selection happens in Project.
          */
         ResourceSelectionChanged,
+        /**
+         * Occurs in Outlook when one or more messages are selected or deselected.
+         * **Important**: Only available with task pane implementation.
```

Comment:
This Important note seems ambiguous to me. Does it mean that the event is only triggered/handled in a JavaScript in a task pane; that is, _not_ in a function command? If not, what exactly is meant?

Reply:
Thanks for the feedback, @Rick-Kirkham. You're correct; the event can only be handled in a task pane and not in a function command. How about something like, "This event can only be handled in a task pane. It isn't supported in function commands."?

Reply:
Sounds great.